portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Sherry's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body>
+    <header id="gallery-head" class="gallery-background">
+      <div id="nav">
+        <ul>
+          <li><a href="gallery.html">Gallery</a></li>
+          <li><a href="index.html#to-do">To-Do's</a></li>
+          <li><a href="index.html#projects">Projects</a></li>
+          <li><a href="index.html#timeline">Timeline</a></li>
+          <li><a href="index.html#about">About Me</a></li>
+          <li><a href="index.html#home">Home</a></li>
+        </ul>  
+      </div>
+      <div class="gallery-title">
+        <h3>My Gallery</h3>
+      </div>
+    </header>
+    </header>
+    <!-- Gallery Section --> 
+    <section id="gallery" class="light"> 
+      <div class="container">
+        <div class="row"> 
+            <div class="column gallery quarter">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+            </div>
+            <div class="column gallery quarter">
+                <img src="/images/sunset.jpg" style="width:100%">
```

Comment:
If you change the img src to lead with a period then the images will work in [HTML preview](https://htmlpreview.github.io/?https://github.com/Sherry2001/step/blob/71ead9ea53dc4a8b431ad39b6785c706a1d1fd66/portfolio/src/main/webapp/index.html):
`<img src="./images/sunset.jpg" alt="a sunset">`

Reply:
ok! changed for all, and will add alt when i fill in the gallery